R/util-MakeBounds.R

```diff
@@ -31,9 +31,19 @@ MakeBounds <- function(
   dSnapshotDate = unique(dfResults$SnapshotDate),
   strStudyID = unique(dfResults$StudyID)
 ) {
-  stop_if_empty(strMetrics)
-  stop_if_empty(dSnapshotDate)
-  stop_if_empty(strStudyID)
+  # stop_if(cnd = !length(strMetrics), message = "`strMetrics` must not be `NULL`.")
+  # stop_if(cnd = !length(dSnapshotDate), message = "`dSnapshotDate` must not be `NULL`.")
+  # stop_if(cnd = !length(strStudyID), message = "`strStudyID` must not be `NULL`.")
```

Comment:
@lauramaxwell @jonthegeek @samussiah @jwildfire which looks better ? The single line call for each stop statement or a big chunked one demo'd below

This would affect all the `stopifnot()` calls too, and realistically all the `if()` then `stop()` calls can be wrapped already with commits here can be wrapped in too

Reply:
One statement per condition feels simpler, and `stop_if` is more intuitive than `stopifnot`.